ffi/diplomat/Cargo.toml

```diff
@@ -142,8 +142,8 @@ unicode-bidi = { version = "0.3.13", default-features = false, optional = true }
 # The version here can either be a `version = ".."` spec or `git = "https://github.com/rust-diplomat/diplomat", rev = ".."`
 # Since this crate is published, Diplomat must be published preceding a new ICU4X release but may use git versions in between
 # ALSO MAKE SURE TO UPDATE /tools/ffi_coverage/Cargo.toml!
-diplomat = { git = "https://github.com/rust-diplomat/diplomat", rev = "8d125999893fedfdf30595e97334c21ec4b18da9" }
-diplomat-runtime = { git = "https://github.com/rust-diplomat/diplomat", rev = "8d125999893fedfdf30595e97334c21ec4b18da9" }
+diplomat = { version = "0.6" }
```

Comment:
we should eventually use workspace dependencies for this and update `diplomat-get-rev`